pkg/windows/windows.go

```diff
@@ -857,15 +864,19 @@ func (vm *windows) ensureHNSNetworksAreRemoved() error {
 	// VIP HNS endpoint created by the operator is also deleted when the HNS networks are deleted.
 	for _, network := range []string{BaseOVNKubeOverlayNetwork, OVNKubeOverlayNetwork} {
 		err = wait.Poll(retry.Interval, retry.Timeout, func() (bool, error) {
+			// retry removing network on failure
```

Comment:
If we keep seeing `read: connection reset by peer` on the retry we might have to call `vm.Reinitialize()`.